app.py

```diff
@@ -231,6 +231,12 @@ def check_language():
 def before_request_begin_logging():
     querylog.begin_global_log_record(path=request.path, method=request.method)
 
+# A context processor injects variables in the context that are available to all templates.
+@app.context_processor
+def enrich_context_with_user_info():
+    user = session.get('user', {'username': '', 'email': ''})
```

Comment:
This object should be standardized somewhere, probably close to the `copy_data_from_database_user` function...

Reply:
As discussed:

We will repurpose the `current_user` function to return session user.